checkov/ansible/graph_builder/local_graph.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+
+import logging
+from copy import deepcopy
+from pathlib import Path
+from typing import Any
+
+from checkov.common.graph.graph_builder import CustomAttributes
+from checkov.common.graph.graph_builder.graph_components.block_types import BlockType
+from checkov.common.graph.graph_builder.graph_components.blocks import Block
+from checkov.common.runners.graph_builder.local_graph import ObjectLocalGraph
+from checkov.common.util.consts import START_LINE, END_LINE
+from checkov.ansible.graph_builder.graph_components.resource_types import ResourceType
+from checkov.ansible.utils import get_scannable_file_paths, TASK_RESERVED_KEYWORDS, parse_file
+
+
+class AnsibleLocalGraph(ObjectLocalGraph):
+    def __init__(self, definitions: dict[str | Path, dict[str, Any] | list[dict[str, Any]]]) -> None:
+        super().__init__(definitions=definitions)
+
+        self.source = "Ansible"
```

Comment:
I think it's time to map the different graph source types into enums, WDYT?

Reply:
yeah, sounds good